charts/argo-cd/README.md

```diff
@@ -64,7 +64,170 @@ applicationSet:
   replicas: 2
 ```
 
-### Synchronizing Changes from Original Repository
+## Ingress configuration
+
+Please refer to [Operator Manual](https://argo-cd.readthedocs.io/en/stable/operator-manual/ingress/#ingress-configurationh) for details as samples below
+corespond to respetive sections.
```

Comment:
>respetive

typo

Reply:
should also probably be "to their respective sections.

Reply:
Fixed